Library/Homebrew/cask/audit.rb

```diff
@@ -528,7 +527,7 @@ def audit_signing
       end
     end
 
-    sig { returns(T.nilable(T.any(T::Boolean, Symbol))) }
+    sig { returns(T.any(NilClass, T::Boolean, Symbol)) }
```

Comment:
This equivalent but marginally more readable imo (one fewer nesting of parens)

Reply:
This sort of weird multi-typing is probably something to address at some point. Cask is littered with it so it's a big task.

There's a _lot_ of nilable boolean stuff that is also very weird in places. It make sense for certain triple-state CLI arguments where explitict yes, explicit no and implicit default intentionally have diferent behaviours. But some other bits I'm less sure it's necessary.